### PR TITLE
Add Docker section to README.md

### PR DESCRIPTION
**Pull request by alexbeatnik**
Delete section Deploy and destroy application
Add section docker to Execution
Add link to docker site

### CODE REVIEW

**Comment by alexbeatnik** [comment]
@extsoft plese review this pull request

File: README.md [hunk 1]
```diff
@@ -76,6 +76,36 @@ If you want to delete vm you must run:
...
+```bash
+docker-compose up
+```
+In this case, the application will be launched on the localhost
+(127.0.0.1)
+
+If you want to run the application on another ip you need to pass the
+parameter with ip:
+```bash
+docker-compose up 'YOUR IP'
```
Comment (extsoft) [inline README.md]: @alexbeatnik is it going to work? I think the command should be like `APP_HOST=asdada docker-compose up`.

File: README.md [hunk 2]
```diff
@@ -76,6 +76,36 @@ If you want to delete vm you must run:
...
+To be able to use this type of run, you need to have
+[Docker](https://www.docker.com/) engine release 18.06.1-ce
+
+A simple way to check Docker:
+```bash
+docker --version
+```
+First of all, you need to run:
+```bash
+docker-compose up
```
Comment (extsoft) [inline README.md]:
```suggestion
docker-compose up -d
```

File: README.md [hunk 3]
```diff
@@ -76,6 +76,36 @@ If you want to delete vm you must run:
...
+If you want to run the application on another ip you need to pass the
+parameter with ip:
+```bash
+docker-compose up 'YOUR IP'
+```
+If you want to shutdown docker you must run:
+```bash
+docker compose-down
+```
+Please note that if you use Linux, then before each command in this section
```
Comment (extsoft) [inline README.md]: @alexbeatnik let's remove it as it depends on how Docker is installed. And, I suppose, people know how to work with `sudo`.

File: README.md [hunk 4]
```diff
@@ -76,6 +76,36 @@ If you want to delete vm you must run:
...
+
+A simple way to check Docker:
+```bash
+docker --version
+```
+First of all, you need to run:
+```bash
+docker-compose up
+```
+In this case, the application will be launched on the localhost
```
Comment (extsoft) [inline README.md]: @alexbeatnik Am I right that if `APP_HOST` is empty,  a `localhost` is used by default.

Comment (alexbeatnik) [reply README.md]: Yes it works like that

**Comment by alexbeatnik** [comment]
@extsoft Can you review again?

File: README.md [hunk 5]
```diff
@@ -76,6 +76,34 @@ If you want to delete vm you must run:
...
+(127.0.0.1)
+
+If you want to run the application on another IP you need to pass the
+parameter with IP:
+```bash
+APP_HOST="IP" docker-compose up -d
+```
+If you want to shutdown docker you must run:
+```bash
+docker compose-down
```
Comment (extsoft) [inline README.md]:
```suggestion
docker-compose down
```

Comment (alexbeatnik) [reply README.md]: @extsoft Done

**Pull request merged**